src/components/subscriptions/hooks/licenseManagerHooks.js

```diff
@@ -0,0 +1,168 @@
+import { useState, useEffect } from 'react';
+import qs from 'query-string';
+
+import LicenseManagerApiService from '../../../data/services/LicenseManagerApiService';
+import NewRelicService from '../../../data/services/NewRelicService';
+
+
+export const useSubscriptions = (includeInActive = false) => {
+  const [subscriptions, setSubscriptions] = useState({
+    results: [],
+    count: 0,
+    next: null,
+    previous: null,
+  });
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState(null);
```

Comment:
I added the initialState argument because PyCharm gives a warning that `useState` expects an argument and none is provided. Also, I think it is a better idea to provide initial state.